tests/unit/commands/local/lib/swagger/test_parser.py

```diff
@@ -11,17 +11,20 @@
 
 
 class TestSwaggerParser_get_apis(TestCase):
+    def setUp(self) -> None:
+        self.stack_path = Mock()
```

Comment:
why do me mock a String?

Reply:
I prefer to mock a string whenever possible because for a string it might "coincidentally get it right" in tests but for `Mock()` object it is guarantee to be unique